tests/js/server/resilience/move/moving-shards-cluster.inc

```diff
@@ -409,8 +409,12 @@ exports.MovingShardsSuite = function({useData, replVersion}) {
     while (true) {
       var job = queryAgencyJob(result.json.id);
       console.info("Status of cleanOutServer job:", job.status);
-      if (job.error === false && job.status === "Finished") {
-        return result;
+      if (job.error === false) {
+        if (job.status === "Finished") {
+          return result;
+        } else if (job.status === "Failed") {
```

Comment:
Is it possible for a job to have `job.error === true && job.status === "Failed"?
Or should the `job.status === "Failed"` check be moved out of the outer `job.error === false` condition here?

Reply:
No, `error === true` is used to indicate that the job was not found. In such case, `job` is `null`.